Z2A SwiftUI/NSCoffee/NSCoffee/Models/Extra.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Extra.swift
+//  NSCoffee
+//
+//  Created by Rob Whitaker on 04/03/2025.
+//
+
+import Foundation
+
+class Extra: Equatable {
```

Comment:
Nice one. That was on my todo list too. Now we can charge for alternative drinks too 😁